self-hosted-login/okta-aspnet-mvc-example/Views/Account/Login.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@{
+    ViewData["Title"] = "Login";
+}
+
+<!-- Latest CDN production Javascript and CSS: 2.6.0 -->
+<script src="https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/2.6.0/js/okta-sign-in.min.js" type="text/javascript"></script>
+<link href="https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/2.6.0/css/okta-sign-in.min.css" type="text/css" rel="stylesheet" />
+
+<h2>Login</h2>
+<div id="widget"></div>
+
+<form method="POST" asp-action="Login">
+    <input type="hidden" name="sessionToken" id="hiddenSessionTokenField" />
+    @Html.AntiForgeryToken()
+</form>
+
+<script type="text/javascript">
+    const signIn = new OktaSignIn({
```

Comment:
I was surprised, there is more browser support for const than I realized! https://caniuse.com/#search=const 👍